lib/MusicBrainz/Server/Data/Role/Collection.pm

```diff
@@ -10,6 +10,9 @@ sub find_by_collection {
     my $table = $self->_table;
     my $type = $self->_type;
 
+    # Is mostly unset, causing a warning when interpolating it below.
+    $extra_join = '' unless defined($extra_join);
```

Comment:
This would normally be written `$extra_join //= '';` I would move it immediately below the line where the variable is set and don’t think this is worth a comment.